app/uk/gov/hmrc/transitmovementsvalidator/models/errors/ErrorCode.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsvalidator.models.errors
+
+/** Common error codes documented in [[https://developer.service.hmrc.gov.uk/api-documentation/docs/reference-guide#errors Developer Hub Reference Guide]]
+  */
+object ErrorCode {
```

Comment:
Where do we use these - could they be a field eg 'name' within the case class?

Reply:
Just copied them from the `transit-movements` project, they are used in the `HttpFormats` trait for creating common error JSON messages on failures. 

Truth be told I'm not 100% happy with how the transit error stuff is done, but it works and I just wanted to get something up to speed. I might tinker to try to consolidate and improve it if I get some time.